### ISSUE TITLE
Tabs accessibility improvements

### ISSUE THREAD

**Issue by desig9stein** [IgniteUI/igniteui-webcomponents#473]
## Description  
The screen reader tells you which item you are on but not the total number of items.
for example, it says that you are on tab one of one. It should say, you are on tab one of five...

**Comment by rkaraivanov** [comment]
@desig9stein

Resurrecting an old issue, but can you test with the current version of the library?

**Comment by desig9stein** [comment]
> @desig9stein
> 
> Resurrecting an old issue, but can you test with the current version of the library?

Same problem, it says that you are on tab one of one.

**Comment by rkaraivanov** [comment]
Is that the native text-to-speech tool in OS or a specialized tool such as NVDA?

**Comment by simeonoff** [comment]
> Is that the native text-to-speech tool in OS or a specialized tool such as NVDA? I've tested with the both NVDA and the native Windows narrator and it does announce correct information.

Speaking for @desig9stein here, but it should be VoiceOver(native to macOS).

**Comment by desig9stein** [comment]
> > Is that the native text-to-speech tool in OS or a specialized tool such as NVDA? I've tested with the both NVDA and the native Windows narrator and it does announce correct information.
> 
> Speaking for @desig9stein here, but it should be VoiceOver(native to macOS).

I tested with the native tool for mac
